hw/top_earlgrey/ip_autogen/pwrmgr/data/pwrmgr_sec_cm_testplan.hjson

```diff
@@ -68,7 +68,7 @@
               tb.dut.rom_ctrl_i
             '''
       stage: V2S
-      tests: ["pwrmgr_sec_cm_rom_ctrl_intersig_mubi"]
+      tests: ["pwrmgr_wakeup_reset", "pwrmgr_repeat_wakeup_reset"]
```

Comment:
Is this intended?

Reply:
Yes it is, per the comment in the seventh commit https://github.com/lowRISC/opentitan/pull/26382/commits/9cbba87c889f8c989e9141c123d7fa9660d781d6.

Reply:
Yes, and it is the seventh commits here, and the comment explains it: https://github.com/lowRISC/opentitan/pull/26382/commits/9cbba87c889f8c989e9141c123d7fa9660d781d6.